rules/rule-credentials-in-env-var/test/pod-sensitive-key-regression/input/input.yml

```
@@ -0,0 +1,19 @@
+apiVersion: v1
```

YAMLlint (1.37.1)

[error] wrong new line character: expected \n

(new-lines)